### ISSUE TITLE
Enhance accessibility for the TOC

### ISSUE THREAD

**Issue by ahmadcisco** [tscanlin/tocbot#387]
Currently, when navigating the generated TOC using the Tab key, only top-level headings (e.g., h2) are reachable. If a heading has subheadings (e.g., h3, h4), these are not reachable via keyboard navigation (Tab key). This creates a frustrating experience for keyboard users and makes the TOC less accessible overall.

I hope you can look into it. Thanks in advance.

**Comment by tscanlin** [comment]
I put up a PR with a fix here:
https://github.com/tscanlin/tocbot/pull/388

**Comment by ahmadcisco** [comment]
Apologies for the late reply, I only just recently was able to fetch the latest version and test this feature. I can confirm that it works. Thank you for taking care of this swiftly!
